aea/aea_builder.py

```diff
@@ -1599,7 +1599,10 @@ def _check_package_dependencies(
         :param configuration: the component configuration
         :raises AEAException: if there's a missing dependency.
         """
-        not_supported_packages = configuration.package_dependencies.difference(
+
+        not_supported_packages = {
+            dep.without_hash() for dep in configuration.package_dependencies
```

Comment:
why without hash?

Reply:
should also be a comment in this code

Reply:
https://github.com/valory-xyz/open-aea/pull/103#discussion_r844255533, I'll add comments 😅